readme.md

```diff
@@ -50,6 +50,14 @@ cd ./cmd/http-reset-server && go run http-reset-server.go
 
 This will expose the stored data through a REST API running on `http://localhost:8080`.
 
+6. Open another terminal in the project root folder and run:
+
+```sh
+cd ./cmd/alert-manager && go run alert-manager.go
+```
+
+This will create send alerts to some clients who are subscribed on the topic `airport/codeIata/alert/#` for all metrics, or for a specific metric like for example the temperature `airport/codeIata/alert/temperature`
+
```

Comment:
```suggestion
This will send alerts to clients who are subscribed on the topic `airport/codeIata/alert/#` for all metrics related to an airport, or for a specific metric like the temperature with `airport/codeIata/alert/temperature`.
```